.circleci/config.yml

```diff
@@ -1,4 +1,6 @@
 version: 2.1
+orbs:
+  win: circleci/windows@2.2.0  
```

Comment:
This uses 2.2.0 but 2.4.0 is the latest. https://circleci.com/developer/orbs/orb/circleci/windows

I'm new to circleci orbs, maybe the version isn't important.  I wasn't able to find the 2.2.0 source to compare them.  Did you come across docs or anything that explains the differences?

Reply:
I couldn't find a history for the version either to see what changed, but using the latest version sounds like a good idea to me.